docs/migration/v1.1.md

```diff
@@ -0,0 +1,129 @@
+# Migrate to v1.1
+
+When updating the SDK from version 1.0.z to 1.1.z you will need to make some changes to your code.
+This document explains the changes required to build your application again with the new SDK
+version.
+
+## Package name changes
+
+The SDK switched to Kotlin Multiplatform starting from v1.1. As a side effect some of our modules
+have been moved around or renamed.
+
+Each module now has separate builds for different platforms (common, jvm and android). Additionally
+the separate jellyfin-platform-android module was merged into jellyfin-core.0
+
+The table below shows the new names to be used:
+
+| Module (before)               | Module (after)    |
+| ----------------------------- | ----------------- |
+| jellyfin-core                 | jellyfin-core     |
+| **jellyfin-platform-android** | **jellyfin-core** |
+| jellyfin-api                  | jellyfin-api      |
+| jellyfin-model                | jellyfin-model    |
+
+Gradle will automatically use the Android build when the Android plugin is enabled.
+If you need to explicitly use the Android build you can add the "-android" suffix.
+
+## SDK initialization
+
+Creating an instance of the SDK got some changes. The `Jellyfin {}` function is now renamed to
+`createJellyfin {}`. The options set inside of this builder changed depending on your platform.
+If the settings are incorrect they will throw an exception on start. So try running your app
+before publishing your changes!
```

Comment:
```suggestion
If the settings are incorrect it will throw an exception on initialization.
So try running your app before publishing your changes!
```

I guess you could also leave away the second sentence..